specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -2303,6 +2303,10 @@
     "CosmosDbMongoDbApiLinkedServiceTypeProperties": {
       "description": "CosmosDB (MongoDB API) linked service properties.",
       "properties": {
+        "isServerVersionAbove32": {
```

Comment:
>isServerVersionAbove32 [](http://example.com/codeflow?start=9&length=22)

Can we have a property like serviceVersion instead?

Reply:
@j5lim In our scenario, only version 3.2 is special, we no need to know the exact server version. If we change it to serverVersion, customer need to select one from a list, which is a more complicated user experience.
And the property name is already used in our backend and UI, I cannot change it easily.